core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -50,6 +50,7 @@ oppia.factory('oppiaPlayerService', [
     var answerIsBeingProcessed = false;
 
     var exploration = null;
+    var manualParamChanges = null;
```

Comment:
Also, it might be nice to indicate here clearly (preferably in the varname?) that this variable only applies during editor preview mode. It took me a while to realize that.